static/js/components/AddToListDialog.js

```diff
@@ -123,22 +123,12 @@ export default function AddToListDialog() {
           </div>
         </div>
       </div>
-      {userLists.map(
-        (userList, i) =>
-          resource &&
-          (![LR_TYPE_LEARNINGPATH, LR_TYPE_USERLIST].includes(
-            resource.object_type
-          ) ||
-            (resource.id !== userList.id &&
-              emptyOrNil(
-                resource.items.filter(
-                  item =>
-                    item.object_id === userList.id &&
-                    [LR_TYPE_LEARNINGPATH, LR_TYPE_USERLIST].includes(
-                      item.content_type
-                    )
-                )
-              ))) ? (
+      {resource &&
+      ![LR_TYPE_LEARNINGPATH, LR_TYPE_USERLIST].includes(
+        resource.object_type
+      ) ? (
```

Comment:
we end up making this check in a bunch of places - what do you think about adding a `isUserList` or similar function in `lib/learning_resources`?

Reply:
Good idea, just pushed that change